runtime/doc/ui.txt

```diff
@@ -228,6 +228,10 @@ the editor.
 	however a UI might still use such options when rendering raw text
 	sent from Nvim, like for |ui-cmdline|.
 
+["chdir", path] ~
```

Comment:
When this is emitted, it will be too late for the UI to ask the Nvim server what its _previous_ CWD was. So maybe this should also include the previous dir?

    ["chdir", prev_cwd, new_cwd]

Reply:
> When this is emitted, it will be too late for the UI to ask the Nvim server what its _previous_ CWD was. So maybe this should also include the previous dir?
> 
> Or the UI could listen to `DirChanged`.
> 
> ```
> ["chdir", prev_cwd, new_cwd]
> ```

Is there a use case you have in mind where that information is needed?

Reply:
No, just thinking about one-way doors. But augmenting events is backward-compatible, and the `DirChanged` event is an alternative. So no problem.